src/yang/ietf-tcp.yang

```diff
@@ -116,7 +117,7 @@ module ietf-tcp {
       must "../enable-ao = 'true'";
       default true;
       description
-        "Include TCP options in MAC calculation.";
+        "Include TCP options in MAC calculation when set to true.";
```

Comment:
Similarly,

"When set to true, TCP options are included in MAC calculation."

Reply:
looks good to me.

Reply:
Fixed